machina/apps/forum/admin.py

```diff
@@ -39,6 +39,32 @@
 PERM_NOT_SET = 'not-set'
 
 
+class AddEditForumForm(forms.ModelForm):
```

Comment:
I agree with the rest of this PR but not with this change. It is true that you can't create a category under an existing category but removing the "Category" forum type choice doesn't seem to be the right solution IMO. What if the initial parent is "Category C" but then you choose to create your forum under another existing forum (or at the root of the tree)? It wouldn't be possible to create a new category in that case.